static/js/sheets/SheetContent.jsx

```diff
@@ -5,8 +5,7 @@ import Component from 'react-class'
 import $  from '../sefaria/sefariaJquery';
 import Sefaria  from '../sefaria/sefaria';
 import {
-  CollectionStatement,
-  InterfaceText, ProfilePic, SheetAuthorStatement, SheetMetaDataBox, SheetTitle,
+  InterfaceText, SheetMetaDataBox,
```

Comment:
It's not clear to me why these modifications are being made here? Should they actually be in some upstream (or downstream) branch?

Reply:
You're right.  These imports should be in a different PR.  Modified that PR and merged it in and now they are gone.